addons/score/spoke/utils.go

```diff
@@ -0,0 +1,52 @@
+package spoke
+
+import (
+	"os"
+	"strconv"
+)
+
+const bitSizeFloat = 64
+
+// getEnvVars returns a map of key values where the keys are passed as a parameter
+// and the values are from environment variables
+func getEnvVars(keys []string) map[string]string {
```

Comment:
name is to generic its extract envVars as map for specific keys `extractEnvAsMap` would be more suitable for this function